protocol.md

```diff
@@ -52,6 +53,11 @@ KEYID           | string | No       | No
     decisions; it may only be used to narrow the selection of possible keys to
     try.
 
+*   CERTIFICATE: Optional, unauthenticated PEM encoded X.509 certificate for the key
```

Comment:
Excuse my ignorance when it comes to certs, but does this field wind up including the entire cert chain needed to verify the signature?

Reply:
Not at all, I was considering being more specific here.

I think we should allow this to be a PEM encoded cert chain that may include (unathenticated) intermediates. This should not include the root -- that should be collected by the client out of band.

As long as the intermediates give a path to the trusted root, then the cert is considered verified.

Reply:
Yes that sounds great!

Reply:
> add support for the full chain

Agreed, I'll update it to specify that this would be a cert chain including the signing cert and intermediates. Just to clarify, explicitly NOT the root?

Reply:
Yep!